zygrader/ui/themes.py

```diff
@@ -0,0 +1,109 @@
+"""Defines all the color pairs and emojis for each theme.
+To add a new theme, use 'curses.init_pair()' with the 256-colors you would like (use constants).
+
+"""
+
+import curses
+from zygrader.config.preferences import THEMES
+
+
+class Theme:
+    def __init__(self):
+        self.__init_colors()
+
+        self.THEME_COLORS = {
```

Comment:
Usually `UPPER_CASE` is reserved for static members of the class, which are defined outside functions. See the `Event` class for examples.

I would make these static since they wouldn't change